### ISSUE TITLE
Create `docs/TERMS.md`

### ISSUE THREAD

**Issue by vuhung16au** [vuhung16au/pytorch-mastery#14]
Create `docs/TERMS.md` and explain the key terms in PyTorch/

Explain the following: 

 batch size
 learning rate
loss function
optimizer
accelerator (such as CUDA, MPS)

---

Do NOT add any other terms.
